pkg/doc/signature/suite/bbsblssignatureproof2020/suite_test.go

```diff
@@ -132,9 +132,18 @@ func (v *testVerifier) Verify(_ *sigverifier.PublicKey, doc, _ []byte) error {
 
 type testKeyResolver struct {
 	publicKey *sigverifier.PublicKey
+	variants  map[string]*sigverifier.PublicKey
 	err       error
 }
 
-func (r *testKeyResolver) Resolve(string) (*sigverifier.PublicKey, error) {
+func (r *testKeyResolver) Resolve(id string) (*sigverifier.PublicKey, error) {
+	if r.err != nil {
+		return nil, r.err
+	}
+
+	if len(r.variants) > 0 {
+		return r.variants[id], nil
+	}
+
```

Comment:
It seems like a bug. How about 
```
if val,ok := r.variants[id];ok{
    return val,nil
}
```

Reply:
`r.variants` could be undefined. If defined, it should contain the target key (as this code is used for tests)